index.js

```diff
@@ -1,18 +1,95 @@
 /***** Deliverable 1 *****/
 const header = document.querySelector("#header")
 console.log("Here's your header:", header)
-
+// Added defer to the HTML Header
 
 /***** Deliverable 2 *****/
-
+header.style.color = "rgb(0, 72, 255)"
+header.style.fontSize = "5rem"
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+const name = document.querySelector("#profile h2")
+name.textContent = `${traveler.name}`
+
+const nickname = document.querySelector("#profile em")
+nickname.textContent = `${traveler.nickname}`
 
+const likes = document.createElement("p")
+likes.setAttribute('id', 'likes')
+name.parentElement.appendChild(likes)
+likes.textContent = `Likes: ${traveler.likes}  (Click to Add Like!)` 
+
+const img = document.querySelector("#profile img")
+img.src = `${traveler.photo}`
+img.alt = `${traveler.name}`
+
+// Can we access data.js because it is loaded in the HTML header before index.js?
 
 /***** Deliverable 4 *****/
+console.log(traveler.animalSightings)
+const animals = document.querySelector("#animals")
+// const ul = document.createElement("ul")
 
+traveler.animalSightings.forEach(sighting => {
+  let li = document.createElement("li")
+  li.setAttribute('data-id', `${sighting.id}`)
+  
+  let p = document.createElement("p")
+  p.textContent = `${sighting.description}`
+  
+  let image = document.createElement("img")
+  image.setAttribute('src',`${sighting.photo}`)
+  image.setAttribute('alt', `${sighting.species}`)
```

Comment:
You can also add 'src' and 'alt' attributes this way:
image.src = sighting.photo
image.alt = sighting.species

You should use '.setAttribute' for non standard HTML attributes. 

No need for string interpolation since 'sighting.photo' and 'sighting.species' are strings.